### DISCUSSION TITLE
Deleting disabled GitHub repository for Violation of ToS

### DISCUSSION THREAD

**Discussion by LokNplay** [community/community#29364] [Other Feature Feedback, Questions, & Ideas] [answered]
I am the owner of [This Repository ](https://github.com/LokNplay/MagiskOnWSA) who's is disabled because of violation of GitHub's ToS. So I want to delete this repo permanently. Any help is appreciated.

**Comment by airtower-luna** [comment, marked as answer]
That's something you'll have to [contact GitHub support](https://support.github.com/request) about, us other users can't help there. Good luck!

**Comment by dxgldotorg** [comment]
Can anybody explain the terms that were allegedly violated?  Was it an excessive use of GitHub Actions, or a legal request from either Microsoft or Google which should have been posted in the [github/dmca](https://github.com/github/dmca) repo istead of just being silently disabled?

**Comment by dxgldotorg** [comment]
They appear to have ignored my request to delete the fork, instead replying with a boilerplate regarding GitHub Actions.
